annotation/tests/test_annotations.py

```diff
@@ -513,3 +515,35 @@ def test_upload_pages_to_minio(moto_s3: boto3.resource):
         mock_upload_json.assert_called_once_with(
             json.dumps(pages[0].dict()), path_to_object, TEST_TENANT, moto_s3
         )
+
+
+def test_update_pages_array():
+    resulting_set = update_pages_array({1, 2, 3}, {4}, {1, 2})
+    assert resulting_set == {3, 4}
+
+
+def test_kafka_message_needed(annotated_doc: AnnotatedDoc):
+    db = Mock()
+    check_if_kafka_message_is_needed(
+        db,
+        annotated_doc,
+        annotated_doc,
+        annotated_doc.job_id,
+        annotated_doc.file_id,
+        TEST_TENANT,
+    )
+    db.assert_not_called
```

Comment:
could you check for the result/raised exceptions? additionally, assert_not_called is a method

Reply:
The function doesn't explicitly state that it raises an exception. 
I can check for the error raised by db.commit(), however, that would just be testing the side_effect of a Mock() object.
Additionally, the return value of the function is None, so I don't have any result to assert.